src/Sulu/Component/Rest/ListBuilder/AbstractListBuilder.php

```diff
@@ -238,8 +238,19 @@ public function search($search)
      */
     public function sort(AbstractFieldDescriptor $fieldDescriptor, $order = self::SORTORDER_ASC)
     {
-        $this->sortField = $fieldDescriptor;
-        $this->sortOrder = $order;
+        $this->sortFields = array($fieldDescriptor);
```

Comment:
i wanted to implement the case of adding multiple sort fields (which was not possible before)

sort() should i.m.o. always overwrite the previously set value.
that's why i introduced addOrderBy